Wordle.Api/Wordle.Api/Services/PlayerService.cs

```diff
@@ -0,0 +1,70 @@
+using Microsoft.EntityFrameworkCore;
+using Wordle.Api.Data;
+using Wordle.Api.Dtos;
+
+namespace Wordle.Api.Services
+{
+    public class PlayerService
+    {
+        private readonly AppDbContext _db;
+
+        public PlayerService(AppDbContext db)
+        {
+            _db = db;
+        }
+
+        public async Task<IEnumerable<PlayerDto>> GetTopTenScores()
+    {
+        int count = 10;
+        var totalCount = await _db.Players.CountAsync(player => player.GameCount > 6);
+        if (count > totalCount)
+        {
+            count = totalCount;
+        }
+        var players = await _db.Players
+            .Where(player => player.GameCount > 6)
```

Comment:
didn't he say that a player with only one game shouldn't show up on the leaderboard?

Reply:
I was under the impression that he said that because he wanted our leaderboard to rank someone with a low amount of average guesses and a high amount of games played, higher than a person with a low amount of average guesses and a low amount of games played. A player shouldn't show up on the leaderboard having only played one game because the ranking system shouldn't favor them, but if there are only 10 people who have played, all 10 should show up, because they are the top ten players, no matter how many games they've played. That was my understanding at least.

Reply:
hrmm.. I like how my implementation uses less variables, and that you've gotta keep your avg low (like golf). But I see your point about ranking "all" players, because they would be top ten if there were less than 10 players.
I tried to think about the case where the one guy has played tons of games, but sucks, vs someone who is very good, but only plays little. In my mind, as long as the person who played little didn't just get lucky, they're a better player than the player who plays more, but poorly, and they should be ranked above the poor player... IDK, i'll change it to player.GameCount / (player.AverageAttempts * 10 + player.AverageSecondsPerGame)), but they still need to play 4 games!